Tools/AP_Bootloader/board_types.txt

```diff
@@ -267,6 +267,7 @@ AP_HW_ORAQF405PRO                    1155
 AP_HW_CBU_StampH743                  1156
 AP_HW_FOXEERF405_V2                  1157
 AP_HW_CSKY405                        1158
+AP_HW_HAKRCF405V2                    1159
```

Comment:
Might be a good idea to create a separate PR which nabs this ID, just in case this PR takes a while to get in.

Reply:
Thank you for your help.